Build failure: wivrn

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

`services.wivrn.enable = true`

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/310511751

### Relevant log output

```console
Running phase: qtPreHook
@nix { "action": "setPhase", "phase": "qtPreHook" }
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/q5ls0vii39sp157vzl83lgr7myapa86c-source
source root is source
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/locale -DCMAKE_INSTALL_LIBEXECDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/libexec -DCMAKE_INSTALL_LIBDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/lib -DCMAKE_INSTALL_DOCDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/doc/WiVRn -DCMAKE_INSTALL_INFODIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/info -DCMAKE_INSTALL_MANDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/man -DCMAKE_INSTALL_INCLUDEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/include -DCMAKE_INSTALL_SBINDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/sbin -DCMAKE_INSTALL_BINDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/bin -DCMAKE_INSTALL_NAME_DIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/lib -DCMAKE_POLICY_DEFAULT_CMP0025=NEW -DCMAKE_FIND_FRAMEWORK=LAST -DCMAKE_STRIP=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/strip -DCMAKE_RANLIB=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/ranlib -DCMAKE_AR=/nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/ar -DCMAKE_C_COMPILER=gcc -DCMAKE_CXX_COMPILER=g++ -DCMAKE_INSTALL_PREFIX=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9 -DWIVRN_USE_NVENC:BOOL=FALSE -DWIVRN_USE_VAAPI:BOOL=TRUE -DWIVRN_USE_VULKAN_ENCODE:BOOL=TRUE -DWIVRN_USE_X264:BOOL=TRUE -DWIVRN_USE_PIPEWIRE:BOOL=TRUE -DWIVRN_USE_PULSEAUDIO:BOOL=TRUE -DWIVRN_FEATURE_STEAMVR_LIGHTHOUSE:BOOL=TRUE -DWIVRN_BUILD_CLIENT:BOOL=FALSE -DWIVRN_BUILD_DASHBOARD:BOOL=TRUE -DWIVRN_CHECK_CAPSYSNICE:BOOL=FALSE -DFETCHCONTENT_FULLY_DISCONNECTED:BOOL=TRUE -DWIVRN_OPENXR_MANIFEST_TYPE:STRING=absolute -DOVR_COMPAT_SEARCH_PATH:STRING=/nix/store/mfj5jcff5a0g0s159k6wp7bjcqdv86wv-xrizer-0.3/lib/xrizer:/nix/store/qajl0hvm7w26c066j2byzsx3q8v5sl9d-opencomposite-1.0.1521/lib/opencomposite -DGIT_DESC:STRING=v25.9 -DFETCHCONTENT_SOURCE_DIR_MONADO:STRING=/nix/store/yx8ypv9flk7bjp21qcx59yfypy6q03mw-source-patched -DQT_OPTIONAL_TOOLS_PATH=/nix/store/qld4r900hwnf2qq2pl6v1811zgpj4i51-qttools-6.10.0 -DKDE_INSTALL_EXECROOTDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9 -DKDE_INSTALL_BINDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/bin -DKDE_INSTALL_SBINDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/sbin -DKDE_INSTALL_LIBDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/lib -DKDE_INSTALL_LIBEXECDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/libexec -DKDE_INSTALL_CMAKEPACKAGEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/lib/cmake -DKDE_INSTALL_QTPLUGINDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/lib/qt-6/plugins -DKDE_INSTALL_PLUGINDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/lib/qt-6/plugins -DKDE_INSTALL_QMLDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/lib/qt-6/qml -DKDE_INSTALL_INCLUDEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/include -DKDE_INSTALL_LOCALSTATEDIR=/var -DKDE_INSTALL_SHAREDSTATEDIR=/com -DKDE_INSTALL_DATAROOTDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share -DKDE_INSTALL_DATADIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share -DKDE_INSTALL_DOCBUNDLEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/doc/HTML -DKDE_INSTALL_KCFGDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/config.kcfg -DKDE_INSTALL_KCONFUPDATEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/kconf_update -DKDE_INSTALL_KAPPTEMPLATESDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/kdevappwizard/templates -DKDE_INSTALL_KFILETEMPLATESDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/kdevfiletemplates/templates -DKDE_INSTALL_KXMLGUIDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/kxmlgui5 -DKDE_INSTALL_KNOTIFYRCDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/knotifications6 -DKDE_INSTALL_ICONDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/icons -DKDE_INSTALL_LOCALEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/locale -DKDE_INSTALL_SOUNDDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/sounds -DKDE_INSTALL_TEMPLATEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/templates -DKDE_INSTALL_WALLPAPERDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/wallpapers -DKDE_INSTALL_APPDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/applications -DKDE_INSTALL_DESKTOPDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/desktop-directories -DKDE_INSTALL_MIMEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/mime/packages -DKDE_INSTALL_METAINFODIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/metainfo -DKDE_INSTALL_QTQCHDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/doc/qch -DKDE_INSTALL_QCHDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/doc/qch -DKDE_INSTALL_MANDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/man -DKDE_INSTALL_INFODIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/info -DKDE_INSTALL_DBUSDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/dbus-1 -DKDE_INSTALL_DBUSINTERFACEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/dbus-1/interfaces -DKDE_INSTALL_DBUSSERVICEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/dbus-1/services -DKDE_INSTALL_DBUSSYSTEMSERVICEDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/dbus-1/system-services -DKDE_INSTALL_SYSCONFDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/etc -DKDE_INSTALL_CONFDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/etc/xdg -DKDE_INSTALL_AUTOSTARTDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/etc/xdg/autostart -DKDE_INSTALL_LOGGINGCATEGORIESDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/qlogging-categories6 -DKDE_INSTALL_SYSTEMDUNITDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/lib/systemd -DKDE_INSTALL_SYSTEMDUSERUNITDIR=/nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/share/systemd/user
-- The C compiler identification is GNU 14.3.0
-- The CXX compiler identification is GNU 14.3.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/gcc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /nix/store/x8mydcgbry214s802nzvy7fdljx404ym-gcc-wrapper-14.3.0/bin/g++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Found PkgConfig: /nix/store/0skggkamysbh0ksc3582lb69wzrpabcg-pkg-config-wrapper-0.29.2/bin/pkg-config (found version "0.29.2")
-- Found Vulkan: /nix/store/x7yh2n6g7km0rmkcx698f3fln05kk5wb-vulkan-loader-1.4.328.0/lib/libvulkan.so (found version "1.4.328") found components: glslangValidator missing components: glslc
-- Found Git: /nix/store/60440kg5c51rrlpinkk2rkg020q5q6kv-git-2.51.0/bin/git (found version "2.51.0")
-- Found OpenSSL: /nix/store/llswcygvgv9x2sa3z6j7i0g5iqqmn5gn-openssl-3.6.0/lib/libcrypto.so (found version "3.6.0")
[33mCMake Warning (dev) at CMakeLists.txt:93 (find_package):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run "cmake
  --help-policy CMP0167" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.
[0m
-- Found Boost: /nix/store/gwsb412i8x720vhadyx9h29z28c63c4p-boost-1.87.0-dev/lib/cmake/Boost-1.87.0/BoostConfig.cmake (found suitable version "1.87.0", minimum required is "1.75.0") found components: locale url
-- Checking for modules 'libavcodec;libavutil;libswscale;libavfilter'
--   Found libavcodec, version 61.19.101
--   Found libavutil, version 59.39.100
--   Found libswscale, version 8.3.100
--   Found libavfilter, version 10.4.100
-- Checking for module 'libdrm'
--   Found libdrm, version 2.4.126
-- Checking for module 'x264'
--   Found x264, version 0.164.x
-- Checking for module 'libsystemd'
--   Found libsystemd, version 258
-- Checking for module 'libpipewire-0.3'
--   Found libpipewire-0.3, version 1.4.9
-- Checking for module 'libpulse'
--   Found libpulse, version 17.0
-- Checking for modules 'avahi-client;avahi-glib'
--   Found avahi-client, version 0.8
--   Found avahi-glib, version 0.8
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Found nlohmann_json: /nix/store/0cvzm03flrngrv5lf8isy9zzr3wx0awh-nlohmann_json-3.12.0/share/cmake/nlohmann_json/nlohmann_jsonConfig.cmake (found version "3.12.0")
-- Checking for modules 'glib-2.0;gio-2.0;gio-unix-2.0'
--   Found glib-2.0, version 2.84.4
--   Found gio-2.0, version 2.84.4
--   Found gio-unix-2.0, version 2.84.4
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Checking for module 'libnotify'
--   Found libnotify, version 0.8.7
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Checking for module 'librsvg-2.0'
--   Found librsvg-2.0, version 2.61.1
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
Package sysprof-capture-4 was not found in the pkg-config search path.
Perhaps you should add the directory containing `sysprof-capture-4.pc'
to the PKG_CONFIG_PATH environment variable
Package 'sysprof-capture-4', required by 'glib-2.0', not found
-- Found LibArchive: /nix/store/nmp4851gz4791xw2rdc85sz8ja5af8yi-libarchive-3.8.1-lib/lib/libarchive.so (found version "3.8.1")
-- Found ZLIB: /nix/store/z55x0q74zldi64iwamqf8wgrm2iza5rk-zlib-1.3.1/lib/libz.so (found version "1.3.1")
-- Found PNG: /nix/store/x23lssxwyj35r3d3y1x12bs3xgg84jz5-libpng-apng-1.6.50/lib/libpng.so (found version "1.6.50")
-- Found Threads: TRUE
-- Performing Test HAVE_STDATOMIC
-- Performing Test HAVE_STDATOMIC - Success
-- Found WrapAtomic: TRUE
-- Found OpenGL: /nix/store/as2rg79vk4z7frb7hy27nmadc3fp1a4j-libglvnd-1.7.0/lib/libOpenGL.so
-- Found WrapOpenGL: TRUE
-- Found WrapVulkanHeaders: /nix/store/icyb0ffm7i4sajzphnf818firkqlx71h-vulkan-headers-1.4.328.0/include
-- Found OpenMP_C: -fopenmp (found version "4.5")
-- Found OpenMP_CXX: -fopenmp (found version "4.5")
-- Found OpenMP: TRUE (found version "4.5")
-- Found KF6Kirigami: /nix/store/5wrscn4y8v878zpfpqwglj80maj4ydpl-kirigami-6.19.0-dev/lib/cmake/KF6Kirigami/KF6KirigamiConfig.cmake (found version "6.19.0")
-- Found Gettext: /nix/store/m5zldis61kjiylqpxf7jaxs0jsmfq6j4-gettext-0.25.1/bin/msgmerge (found version "0.25.1")
-- Found KF6I18n: /nix/store/dajqzw4ps5k3qqgvf0z7rnd0mkmcixva-ki18n-6.19.0-dev/lib/cmake/KF6I18n/KF6I18nConfig.cmake (found version "6.19.0")
-- Found KF6CoreAddons: /nix/store/yp7xm3zdszimxxnvvc7g24cqa43nsg97-kcoreaddons-6.19.0-dev/lib/cmake/KF6CoreAddons/KF6CoreAddonsConfig.cmake (found version "6.19.0")
-- Found KF6QQC2DesktopStyle: /nix/store/mvrr0czr241wn60q2dj6sb7w6170igbr-qqc2-desktop-style-6.19.0-dev/lib/cmake/KF6QQC2DesktopStyle/KF6QQC2DesktopStyleConfig.cmake (found version "6.19.0")
-- Found KF6IconThemes: /nix/store/ln8dlwp3rkf8v0vspj3cxg7kp8p3fh58-kiconthemes-6.19.0-dev/lib/cmake/KF6IconThemes/KF6IconThemesConfig.cmake (found version "6.19.0")
-- Found KF6: success (found version "6.19.0") found components: Kirigami I18n CoreAddons QQC2DesktopStyle IconThemes
-- Found Python3: /nix/store/cfapjd2rvqrpry4grb0kljnp8bvnvfxz-python3-3.13.8/bin/python3.13 (found version "3.13.8") found components: Interpreter
-- Could NOT find HIDAPI (missing: HIDAPI_INCLUDE_DIR HIDAPI_LIBRARY) 
-- Could NOT find bluetooth (missing: bluetooth_LIBRARY bluetooth_INCLUDE_DIR) 
-- Could NOT find OpenHMD (missing: OPENHMD_INCLUDE_DIR OPENHMD_LIBRARY) 
-- Could NOT find OpenCV (missing: OpenCV_DIR)
-- Could NOT find Libusb1 (missing: LIBUSB1_LIBRARY LIBUSB1_INCLUDE_DIR) 
-- Found JPEG: /nix/store/gdfk1rph83bvw311jq02br1zy2krcmaf-libjpeg-turbo-3.1.2/lib/libjpeg.so (found version "62")
-- Could NOT find realsense2 (missing: realsense2_DIR)
-- Could NOT find depthai (missing: depthai_DIR)
-- Could NOT find SDL2 (missing: SDL2_DIR)
-- Could NOT find cJSON (missing: CJSON_INCLUDE_DIR CJSON_LIBRARY) 
-- Could NOT find LeapV2 (missing: LeapV2_INCLUDE_DIR LeapV2_LIBRARY) 
-- Could NOT find LeapSDK (missing: LeapSDK_DIR)
-- Found ONNXRuntime: /nix/store/5z36lrf98xrwam8zhyj4f9r6w8m5ibdi-onnxruntime-1.22.2-dev/include/onnxruntime
-- Performing Test HAVE_EGL
-- Performing Test HAVE_EGL - Success
-- Found EGL: /nix/store/zxd9sjrf10s65hfxrb08027bajplg5ks-libglvnd-1.7.0-dev/include (found version "1.5")
-- Could NOT find Percetto (missing: Percetto_INCLUDE_DIR Percetto_LIBRARY) 
-- Found Systemd: /nix/store/31x2j1zsjl2cyq6vx9bvfc1y5169ql21-systemd-258-dev/include
-- Found OpenGLES: /nix/store/as2rg79vk4z7frb7hy27nmadc3fp1a4j-libglvnd-1.7.0/lib/libGLESv2.so  found components: V3
-- Checking for module 'libuvc'
--   No package 'libuvc' found
-- Found X11: /nix/store/nml7jra2yzxg133ssim80aj39f4668gc-xorgproto-2024.1/include
-- Looking for XOpenDisplay in /nix/store/0bhv7asksrwac04m9vjiqs2hdrm2jrc4-libx11-1.8.12/lib/libX11.so;/nix/store/pxamnydq8sd9jkrdwqwbl6viaz8lh3p9-libxext-1.3.6/lib/libXext.so
-- Looking for XOpenDisplay in /nix/store/0bhv7asksrwac04m9vjiqs2hdrm2jrc4-libx11-1.8.12/lib/libX11.so;/nix/store/pxamnydq8sd9jkrdwqwbl6viaz8lh3p9-libxext-1.3.6/lib/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Found udev: /nix/store/f8plklbbq3gwkq1wfq89i3f4wy8rabzn-systemd-258/lib/libudev.so
-- Checking for module 'xrandr'
--   Found xrandr, version 1.5.4
-- Checking for modules 'xcb;xcb-randr;x11-xcb'
--   Found xcb, version 1.17.0
--   Found xcb-randr, version 1.17.0
--   Found x11-xcb, version 1.8.12
-- Checking for one of the modules 'wayland-client'
-- Checking for one of the modules 'wayland-scanner'
-- Checking for one of the modules 'wayland-protocols'
-- Found OpenGL: /nix/store/as2rg79vk4z7frb7hy27nmadc3fp1a4j-libglvnd-1.7.0/lib/libOpenGL.so  found components: GLX
-- Checking for one of the modules 'dbus-1'
-- Checking for one of the modules 'libbsd'
-- Checking for modules 'gstreamer-1.0;gstreamer-app-1.0;gstreamer-video-1.0'
--   No package 'gstreamer-1.0' found
--   No package 'gstreamer-app-1.0' found
--   No package 'gstreamer-video-1.0' found
-- Checking for module 'survive'
--   No package 'survive' found
-- OpenVR not found
-- Enabled drivers: drv_cemu drv_multi rift steamvr_lh v4l2
-- Could NOT find Doxygen (missing: DOXYGEN_EXECUTABLE) 
-- #####----- Config -----#####
-- #    GIT_DESC:        v25.9
-- #
-- #    BLUETOOTH:       OFF
-- #    D3D11:           OFF
-- #    D3D12:           OFF
-- #    DBUS:            OFF
-- #    EGL:             ON
-- #    GST (GStreamer): OFF
-- #    HIDAPI:          OFF
-- #    JPEG:            OFF
-- #    LIBBSD:          OFF
-- #    LIBUSB:          OFF
-- #    LIBUVC:          OFF
-- #    ONNXRUNTIME:     OFF
-- #    OPENCV:          OFF
-- #    OPENGL:          ON
-- #    OPENGLES:        ON
-- #    OPENGL_GLX:      ON
-- #    PERCETTO:        OFF
-- #    REALSENSE:       OFF
-- #    OPENVR:          OFF
-- #    SDL2:            OFF
-- #    SYSTEM_CJSON:    OFF
-- #    SYSTEMD:         ON
-- #    TRACY:           OFF
-- #    VULKAN:          ON
-- #    WAYLAND:         ON
-- #    WAYLAND_DIRECT:  ON
-- #    XCB:             ON
-- #    XLIB:            ON
-- #    XRANDR:          ON
-- #
-- #    MODULE_AUX_VIVE:             ON
-- #    MODULE_COMPOSITOR:           ON
-- #    MODULE_COMPOSITOR_MAIN:      ON
-- #    MODULE_COMPOSITOR_NULL:      ON
-- #    MODULE_MERCURY_HANDTRACKING: OFF
-- #    MODULE_IPC:                  ON
-- #    MODULE_MONADO_GUI:           OFF
-- #    MODULE_MONADO_CLI:           ON
-- #    MODULE_OPENXR_STATE_TRACKER: ON
-- #
-- #    FEATURE_AHARDWARE_BUFFER:                      OFF
-- #    FEATURE_CLIENT_DEBUG_GUI:                      OFF
-- #    FEATURE_COLOR_LOG:                             ON
-- #    FEATURE_DEBUG_GUI:                             OFF
-- #    FEATURE_OPENXR:                                ON
-- #    FEATURE_OPENXR_ACTIVE_ACTION_SET_PRIORITY:     ON
-- #    FEATURE_OPENXR_BODY_TRACKING_FB:               OFF
-- #    FEATURE_OPENXR_BODY_TRACKING_FULL_BODY_META:   OFF
-- #    FEATURE_OPENXR_FACE_TRACKING2_FB:              ON
-- #    FEATURE_OPENXR_DEBUG_UTILS:                    OFF
-- #    FEATURE_OPENXR_DISPLAY_REFRESH_RATE:           ON
-- #    FEATURE_OPENXR_FACIAL_TRACKING_HTC:            ON
-- #    FEATURE_OPENXR_FORCE_FEEDBACK_CURL:            ON
-- #    FEATURE_OPENXR_HAND_TRACKING_EXT:              ON
-- #    FEATURE_OPENXR_HAND_TRACKING_DATA_SOURCE_EXT:  ON
-- #    FEATURE_OPENXR_HAPTIC_PCM:                     OFF
-- #    FEATURE_OPENXR_HEADLESS:                       ON
-- #    FEATURE_OPENXR_INTERACTION_EXT_EYE_GAZE:       ON
-- #    FEATURE_OPENXR_INTERACTION_EXT_HAND:           ON
-- #    FEATURE_OPENXR_INTERACTION_EXT_PALM_POSE:      ON
-- #    FEATURE_OPENXR_INTERACTION_ML2:                ON
-- #    FEATURE_OPENXR_INTERACTION_MNDX:               ON
-- #    FEATURE_OPENXR_INTERACTION_MSFT_HAND:          OFF
-- #    FEATURE_OPENXR_INTERACTION_OPPO:               ON
-- #    FEATURE_OPENXR_INTERACTION_WINMR:              ON
-- #    FEATURE_OPENXR_LAYER_COLOR_SCALE_BIAS:         ON
-- #    FEATURE_OPENXR_LAYER_CUBE:                     OFF
-- #    FEATURE_OPENXR_LAYER_CYLINDER:                 ON
-- #    FEATURE_OPENXR_LAYER_DEPTH:                    ON
-- #    FEATURE_OPENXR_LAYER_EQUIRECT1:                OFF
-- #    FEATURE_OPENXR_LAYER_EQUIRECT2:                ON
-- #    FEATURE_OPENXR_LAYER_FB_ALPHA_BLEND:           OFF
-- #    FEATURE_OPENXR_LAYER_FB_DEPTH_TEST:            OFF
-- #    FEATURE_OPENXR_LAYER_FB_IMAGE_LAYOUT           OFF
-- #    FEATURE_OPENXR_LAYER_FB_SETTINGS:              OFF
-- #    FEATURE_OPENXR_LAYER_FB_PASSTHROUGH:           OFF
-- #    FEATURE_OPENXR_OVERLAY:                        ON
-- #    FEATURE_OPENXR_PERFORMANCE_SETTINGS:           OFF
-- #    FEATURE_OPENXR_PLANE_DETECTION:                OFF
-- #    FEATURE_OPENXR_USER_PRESENCE:                  ON
-- #    FEATURE_OPENXR_SPACE_LOCAL_FLOOR:              ON
-- #    FEATURE_OPENXR_SPACE_UNBOUNDED:                ON
-- #    FEATURE_OPENXR_VISIBILITY_MASK:                ON
-- #    FEATURE_OPENXR_VULKAN_SWAPCHAIN_FORMAT_LIST:   ON
-- #    FEATURE_OPENXR_XDEV_SPACE:                     ON
-- #    FEATURE_RENDERDOC:                             OFF
-- #    FEATURE_SERVICE:                               ON
-- #    FEATURE_SERVICE_SYSTEMD:                       OFF
-- #    FEATURE_SLAM:                                  OFF
-- #    FEATURE_SSE2:                                  ON
-- #    FEATURE_STEAMVR_PLUGIN:                        OFF
-- #    FEATURE_TRACING:                               OFF
-- #    FEATURE_WINDOW_PEEK:                           OFF
-- #
-- #    DRIVER_ANDROID:              OFF
-- #    DRIVER_ARDUINO:              OFF
-- #    DRIVER_DAYDREAM:             OFF
-- #    DRIVER_DEPTHAI:              OFF
-- #    DRIVER_EUROC:                OFF
-- #    DRIVER_HANDTRACKING:         OFF
-- #    DRIVER_HDK:                  OFF
-- #    DRIVER_HYDRA:                OFF
-- #    DRIVER_ILLIXR:               OFF
-- #    DRIVER_NS:                   OFF
-- #    DRIVER_OHMD:                 OFF
-- #    DRIVER_OPENGLOVES:           OFF
-- #    DRIVER_PSMV:                 OFF
-- #    DRIVER_PSSENSE:              OFF
-- #    DRIVER_PSVR:                 OFF
-- #    DRIVER_QWERTY:               OFF
-- #    DRIVER_REALSENSE:            OFF
-- #    DRIVER_REMOTE:               OFF
-- #    DRIVER_RIFT:                 ON
-- #    DRIVER_RIFT_S:               OFF
-- #    DRIVER_ROKID:                OFF
-- #    DRIVER_SIMULATED:            OFF
-- #    DRIVER_SIMULAVR:             OFF
-- #    DRIVER_SURVIVE:              OFF
-- #    DRIVER_TWRAP:                OFF
-- #    DRIVER_ULV2:                 OFF
-- #    DRIVER_ULV5:                 OFF
-- #    DRIVER_VF:                   OFF
-- #    DRIVER_VIVE:                 OFF
-- #    DRIVER_WMR:                  OFF
-- #    DRIVER_XREAL_AIR:            OFF
-- #    DRIVER_STEAMVR_LIGHTHOUSE:   TRUE
-- #####----- Config -----#####
[0mSelected encoders:[0m
[0m	NVENC : FALSE[0m
[0m	VAAPI : TRUE[0m
[0m	Vulkan: TRUE[0m
[0m	x264  : TRUE[0m
[0m[0m
[0mAudio backends:[0m
[0m	Pipewire  : TRUE[0m
[0m	Pulseaudio: TRUE[0m
[0m[0m
[0mOptional features:[0m
[0m	systemd: ON[0m
[0mInstalling in /nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9. Run /build/source/build/dashboard/prefix.sh to set the environment for WiVRn.[0m
fatal: not a git repository (or any of the parent directories): .git
fatal: not a git repository (or any of the parent directories): .git
[33mCMake Warning at build/dashboard/Findorg.kde.kirigami-QMLModule.cmake:23 (message):
  qmldir not found in
  /nix/store/dry7rpdpzx7s5wqcvxa4hmh5gr7wg2dk-wivrn-25.9/lib/qt-6/qml/org/kde/kirigami
  or
  /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/qt-6/qml/org/kde/kirigami/qmldir
Call Stack (most recent call first):
  /nix/store/fsl6ljm6689am778f19295qqvkg3rw8r-extra-cmake-modules-6.19.0/share/ECM/modules/ECMFindQmlModule.cmake:56 (find_package)
  dashboard/CMakeLists.txt:7 (ecm_find_qmlmodule)

[0m
-- Could NOT find org.kde.kirigami-QMLModule (missing: org.kde.kirigami-QMLModule_FOUND) 
[33mCMake Warning (dev) at /nix/store/r7wdw4nkkjii690wwnzkgj68cg4shh2v-qtbase-6.10.0/lib/cmake/Qt6Core/Qt6CoreMacros.cmake:3472 (message):
  Qt policy QTP0004 is not set: You need qmldir files for each extra
  directory that contains .qml files for your module.  Check
  https://doc.qt.io/qt-6/qt-cmake-policy-qtp0004.html for policy details.
  Use the qt_policy command to set the policy and suppress this warning.

Call Stack (most recent call first):
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:3897 (__qt_internal_setup_policy)
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:960 (qt6_target_qml_sources)
  /nix/store/rz0lc8irp5knfzwakm86p3ifkflr1w1i-qtdeclarative-6.10.0/lib/cmake/Qt6Qml/Qt6QmlMacros.cmake:1405 (qt6_add_qml_module)
  dashboard/CMakeLists.txt:28 (qt_add_qml_module)
This warning is for project developers.  Use -Wno-dev to suppress it.
[0m
-- Configuring done (34.3s)
[31mCMake Error at /nix/store/r5kzs0r6nlbh8pyaj5vzq8yacvq7l6nm-qcoro-0.12.0-dev/lib/cmake/QCoro6Qml/QCoro6QmlTargets.cmake:55 (set_target_properties):
  The link interface of target "QCoro6::Qml" contains:

    Qt6::QmlPrivate

  but the target was not found.  Possible reasons include:

    * There is a typo in the target name.
    * A find_package call is missing for an IMPORTED target.
    * An ALIAS target is missing.

Call Stack (most recent call first):
  /nix/store/r5kzs0r6nlbh8pyaj5vzq8yacvq7l6nm-qcoro-0.12.0-dev/lib/cmake/QCoro6Qml/QCoro6QmlConfig.cmake:34 (include)
  /nix/store/r5kzs0r6nlbh8pyaj5vzq8yacvq7l6nm-qcoro-0.12.0-dev/lib/cmake/QCoro6/QCoro6Config.cmake:25 (find_package)
  CMakeLists.txt:159 (find_package)

[0m
-- Generating done (6.1s)
[33mCMake Warning:
  Manually-specified variables were not used by the project:

    CMAKE_EXPORT_NO_PACKAGE_REGISTRY
    CMAKE_POLICY_DEFAULT_CMP0025
    FETCHCONTENT_FULLY_DISCONNECTED
    QT_OPTIONAL_TOOLS_PATH

[0m
[0mCMake Generate step failed.  Build files cannot be regenerated correctly.[0m
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.17.3, NixOS, 25.11 (Xantusia), 25.11.20251019.5e2a59a`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.2`
 - channels(root): `"nixos"`
 - nixpkgs: `/nix/store/7agp54mgffm9m1wc1kgmkm37pvy18qhf-source`

### Notify maintainers

@ImSapphire @PassiveLemon

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/wi/wivrn/package.nix
+++ b/pkgs/by-name/wi/wivrn/package.nix
@@ -90,6 +90,15 @@ stdenv.mkDerivation (finalAttrs: {
     fi
   '';
 
+  patches = [
+    # See https://github.com/WiVRn/WiVRn/pull/557
+    (fetchpatch {
+      name = "wivrn-fix-qt6.10-build.patch";
+      url = "https://github.com/WiVRn/WiVRn/commit/2204fdd39682cfc052556d58fdb9404dd8ecf63f.patch?full_index=1";
+      hash = "sha256-05MLfJNCznBt6eaggUfSk1jaNDB2/eou6CfexUkIHZE=";
+    })
+  ];
+
   nativeBuildInputs = [
     cmake
     git

